README.md

```diff
@@ -1,3 +1,59 @@
 # Data Models
 
 Data models and vocabularies in the biomedical space.
+
+## Persistent CSV Format
+
+Data model descriptions are stored persistently in this repository in csv format for portability and human readability. Each data model has its own directory with versions of the model in subdirectories. Each version has at least `definitions` and `schema` directories and, optionally, a `constraints` directory and `indexes.csv` and `references.csv` files.
+
+The `definitions` directory holds basic information about the data model that would be of primary interest to a data user. There is a `tables.csv` file, which lists table names and descriptions, as well as a csv file for each table, which lists field names and descriptions, whether the field is "required" (a governance- not data-level attribute), and optionally a table/field combination to which the field refers (typically manifested as a foreign key relationship).
+
+The `schema` directory holds detailed information that might be used to instantiate the data model in a database or other physical storage medium. There is a csv file for each table that lists `type`, `length`, `precision`, `scale`, and `default` attributes (all optional except `type`) for each field.
+
+The `constraints` directory, if present, can hold any number of csv files which list data level constraints that should be applied to any physical representation of the data model. These files contain a `type`, an optional `name`, and the target `table` and `field` for each constraint.
+
+The `indexes.csv` file, if present, lists indexes that should be built on a physical representation of the data model, with `name`, whether the index should be `unique`, target `table` and `field`, and `order` attributes for each index.
+
+The `references.csv` file, if present, lists references (usually foreign keys) which should be enforced on the data model. Each reference is listed with the source `table` and `field`, the target `table` and `field`, and an optional `name`.
+
+Additionally, each version directory has a `datamodel.json` file that holds metadata about the datamodel and version, so as not to rely on directory structure for interpretability. Finally, each data model root directory may have a `renamings.csv` file that maps fields which have been renamed across versions by providing a source data model `version`, `table`, and `field` and a target `version`, `table`, and `field`.
+
+The top-level `mappings` directory holds a series of csv files which list field level mappings between data models. The files contain a target `model`, `version`, `table`, and `field` as well as a source `model`, `version`, `table`, and `field` along with a free text `comment` for each mapping.
```

Comment:
Again the location of the directory doesn't really matter. Mapping files are sought out in the entire repository. I suggest adding (in parentheses.. no pun intended) the prefix for each set of fields; `source_` and `target_`.